packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -164,6 +197,38 @@ export class IosSimulatorDevice extends DeviceBase {
     ]);
   }
 
+  async changeLocale(newLocale: Locale): Promise<boolean> {
+    const deviceSetLocation = getOrCreateDeviceSet();
+    const languageCode = newLocale.match(/([^_-]*)/)![1];
+    await exec("xcrun", [
+      "simctl",
+      "--set",
+      deviceSetLocation,
+      "spawn",
+      this.deviceUDID,
+      "defaults",
```

Comment:
I wonder if we could simplify the process and avoid restarts if we use the same method for updating defaults as we do for things like schemeapproval list (see below)

Reply:
Unfortunately no, we could change the way we modify relavent, plist but we still have no way to modify what operating system keeps in memory at runtime.

Reply:
ok, but we could still update that before the boot. Now we boot the device to only update this setting and the reboot.

Reply:
you were right I was able to avoid reset on on boot